src/constants.ts

```diff
@@ -104,7 +104,8 @@ export const CUM_EXP_TABLE = EXP_TABLE.map(
 
 export const VOTE_BONUS_DURATION = 1;
 export const VOTE_LINK = "https://top.gg/bot/508759831755096074/vote";
-export const REVIEW_LINK = "https://top.gg/bot/508759831755096074#reviews";
+export const REVIEW_LINK =
```

Comment:
Does this work across all major browsers? Test on non-chromium

Reply:
This will only work in Chromium-based browsers. Top.gg doesn't support an official link to reviews anymore, so this might be our best bet